configuration/app-settings.md

```diff
@@ -102,6 +101,63 @@ Under the registrations key in app_settings, we can setup triggers for scheduled
 |`event[].bool_expr`|A JavaScript expression that will be cast to boolean to qualify execution of the event. Leaving blank will default to always true. CouchDB document fields can be accessed using `doc.key.subkey`. Regular expressions can be tested using `pattern.test(value)` e.g. /^[0-9]+$/.test(doc.fields.last_menstrual_period). In our example above, we're making sure the form has an LMP date.|no|
+|`validations`|A set of validations to do for incoming reports. <!-- TODO: Flesh this out-->|no|
+|`messages`|A set of automated responses to incoming reports. <!-- TODO: Flesh this out-->.|no|
+
+#### Patient Reports
+
+Under the `patient_reports` key in app_settings, we can setup actions to take for other form submissions. A above can be defined as shown below:
+
+```json
+  "patient_reports": [
+    {
+    {
+      "form": "V",
+      "name": "Visit (SMS)",
+      "format": "V <patientid>",
+      "silence_type": "ANC Reminders, ANC Reminders LMP, ANC Reminders LMP from App",
+      "silence_for": "8 days",
+      "fields": [
+        {
+          "field_name": "",
+          "title": ""
+        }
+      ],
+      "validations": {
+        "join_responses": true,
+        "list": [
+          {
+            "property": "patient_id",
+            "rule": "regex('^[0-9]{5,13}$')",
+            "translation_key": "messages.generic.validation.patient_id"
+          }
+        ]
+      },
+      "messages": [
+        {
+          "translation_key": "messages.v.report_accepted",
+          "event_type": "report_accepted",
+          "recipient": "reporting_unit"
+        },
+        {
+          "translation_key": "messages.generic.registration_not_found",
+          "event_type": "registration_not_found",
+          "recipient": "reporting_unit"
+        }
+      ]
+    },
+```
+
+|property|description|required|
+|-------|---------|----------|
+|`form`|Form ID of the patient form.|yes|
+|`name`|Descriptive name of the form. This is not currently used in the app, but can be a helpful annotation.|no|
+|`format`|Guide of how the form can be used. This is not currently used in the app, but can be a helpful annotation.|no|
+|`silence_type`|A comma separated list of schedules to mute.|no|
+|`silence_for`|Duration from when the report was submitted for which messages should be muted. It is structured as a string with an integer value followed by a space and the time unit. For instance `8 weeks` or `2 days`. The units available are `seconds`, `minutes`, `hours`, `days`, `weeks`, `months`, `years`, and their singular forms as well. When a message is muted all messages belonging to the same group will be muted, even if it falls outside of this time period. See `messages[].group` in _Schedules_ for related info.|no|
+|`fields`|Descriptive list of form fields. This is not currently used in the app, but can be a helpful annotation.|no|
+|`validations`|A set of validations to do for incoming reports. <!-- TODO: Flesh this out-->|no|
```

Comment:
@eljhkrr it would be helpful if you could complete this as well, repeat of the fields from above. Thanks!